### PR TITLE
Summary page changes

### PR DESCRIPTION
**Pull request by manoj22-k**
In Mtool Frontend updated UI based on internal review, showed telemetry metrics and made UI responsive.
In MTool Backend updated apis to get metrics from telemetry db.
image
image

### CODE REVIEW

**Review by mjlee34 — APPROVED** [review]
LGTM

File: src/mtool/api/rest/rest_api/dagent/metrics.py [hunk 1]
```diff
@@ -251,15 +238,11 @@ def get_vol_read_latency(time, arr_ids, vol_ids, auth = BASIC_AUTH_TOKEN):
...
     try:
-        #print("Sending command to dagent")
-        PATH = ARRAY_PATH.format("writelatency",arr_ids,time)
-        response = send_command_to_dagent("GET",url = PATH, headers= req_headers,timeout=(connect_timeout,read_timeout))
-        #print("--------------RESPONSE-------------")
-        response = response.json()
-        #print("--------------RESPONSE-------------")
+        PATH = "{metric_query_path}{walv}".format(metric_query_path=metric_query_path, walv=WRITE_AVG_LAT_VOLUME)
+        response = requests.get(PATH)
+        response = json.loads(response.content)
```
Comment (vishal-shakya-ssir) [inline src/mtool/api/rest/rest_api/dagent/metrics.py:245]: Please check the scenario if Telemetry is not running then how requests.get() and json.loads() will behave. 
Will this function return meaningful error message to UI ?

File: src/mtool/api/rest/rest_api/perf/system_perf.py [hunk 2]
```diff
@@ -131,40 +94,55 @@ def get_perf_from_influx(query):
...
-        len(latency_res["result"]["data"][0]) > 0:
-        latency = latency_res["result"]["data"][0][0]["latency"]
-    res_dict['latency'] = latency
+    read_latency = 0
+    write_latency = 0
+    read_latency_res = metrics.get_read_latency(metric_query_path)
+    if read_latency_res is not None and "data" in read_latency_res and "result" in read_latency_res["data"] and len(read_latency_res["data"]["result"]) > 0:
+        for data in read_latency_res["data"]["result"]:
+            if "value" in data:
+                read_latency += int(data["value"][1])
```
Comment (vishal-shakya-ssir) [inline src/mtool/api/rest/rest_api/perf/system_perf.py:136]: Will data["value"] have index 1 always ? Any possibly  of data["value"] does not have index 1 ?

**Review by vishal-shakya-ssir — CHANGES_REQUESTED** [review]

File: src/mtool/api/rest/app.py [hunk 3]
```diff
@@ -792,9 +792,14 @@ def get_volume_write_latency():
...
 @app.route('/api/v1/perf/all', methods=['GET'])
 def get_current_iops():
-    array_ids = "0,1"
-    res = get_disk_current_perf(array_ids)
-    return jsonify(res)
+    try:
+        received_telemetry = connection_factory.get_telemetery_url()
+        ip = received_telemetry[0]
+        port= received_telemetry[1]
+        res = get_disk_current_perf(ip, port)
```
[the remark this answers is not in the document]
Comment (manoj22-k) [reply src/mtool/api/rest/app.py]: Changed get_disk_current_perf function name to get_agg_volumes_perf, thanks

File: src/mtool/api/rest/rest_api/dagent/metrics.py (hunk 1 above)
Comment (manoj22-k) [reply src/mtool/api/rest/rest_api/dagent/metrics.py:245]: Added is Telemetry Endpoint up check when loading dashboard page, showed alert popup if not running with meaningful error message.

File: src/mtool/api/rest/rest_api/perf/system_perf.py (hunk 2 above)
Comment (manoj22-k) [reply src/mtool/api/rest/rest_api/perf/system_perf.py]: Yes, though value should have index 1, added check if it has index 1 then only add. Thank you.

**Review by vishal-shakya-ssir — APPROVED** [review]
Thanks for making changes of review comments 
LGTM

File: src/mtool/ui/src/sagas/dashboardSaga.js [hunk 4]
```diff
@@ -91,11 +92,27 @@ export function* fetchAlertsInfo() {
 }
 */
 
+
+export function* fetchCheckTelemetry() {
+  try {
+    const response = yield call([axios, axios.get], '/api/v1/checktelemetry');
+    if (response.status === 200 && response.data) {
+      if (response.data.isTelemetryEndpointUp === true)
+        yield put(actionCreators.setShowTelemetryNotRunning(false, ""));
```
[the remark this answers is not in the document]
Comment (manoj22-k) [reply src/mtool/ui/src/sagas/dashboardSaga.js:101]: If we load this page and telemetry api is not up or prometheus db is not running , we will show alert popup with meaningful error message.

File: src/mtool/api/rest/rest_api/perf/system_perf.py [hunk 5]
```diff
@@ -131,40 +94,40 @@ def get_perf_from_influx(query):
     return val
 """
 
-def get_disk_current_perf(array_ids):
+def get_agg_value(response):
+    value = 0
+    if response is not None and "data" in response and "result" in response["data"] and len(response["data"]["result"]) > 0:
+        for data in response["data"]["result"]:
+            if "metric" in data and "job" in data["metric"] and data["metric"]["job"] == "pos" and "value" in data and len(data["value"]) == 2:
+                value += int(data["value"][1])
```
[the remark this answers is not in the document]
Comment (manoj22-k) [reply src/mtool/api/rest/rest_api/perf/system_perf.py]: Used aggregated function sum, Thanks for the review.

File: src/mtool/ui/src/sagas/dashboardSaga.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (aswinssir) [reply src/mtool/ui/src/sagas/dashboardSaga.js:101]: Okay

**Review by aswinssir — APPROVED** [review]

**Pull request merged**